packages/flutter/lib/src/widgets/title.dart

```diff
@@ -18,6 +18,7 @@ class Title extends StatelessWidget {
   Title({
     super.key,
     this.title = '',
+    this.automaticAppSwitcherDescAdjustment = true,
```

Comment:
The whole purpose of the Title widget is to populate the ApplicationSwitcher with the title. If you don't want that, you might as well just not use the Title widget.